addons/mass_mailing/views/mass_mailing_views.xml

```diff
@@ -591,10 +591,8 @@
                         </div>
 
                         <group>
-                            <label for="name" string="Subject"/>
-                            <div>
-                                <field name="name" nolabel="1"/>
-                            </div>
+                            <field name="name" required="1" string="Mailing Name"/>
```

Comment:
Isn't the name already required ? (as it came from utm.source)

Reply:
To my surprise it was not.